examples/kubeval/simple/README.md

```diff
@@ -1,31 +1,30 @@
-# Kubeval
+# kubeval: simple example
```

Comment:
Can you also uncomment the function in `nginx` package and test it works in the quickstart docs:

https://kpt.dev/book/01-getting-started/02-quickstart

Things to test:
- It passes with no change to package
- It fails if user introduced bad changes.

Reply:
kubeval works fine in the quickstart.